### PR TITLE
Add mtucker's Boston file

### PR DESCRIPTION
**Pull request by markrtucker**
Added a few places to get great food in Boston.

### CODE REVIEW

**Comment by githubteacher** [comment]
@mtucker01752 do you have any suggestions for places to find awesome local 🍻 ?

Hooray Boston!!! ✨ 
i

**Comment by markrtucker** [comment]
@githubteacher Tavern in the Square has lots of local micro-brews on tap. Or check out Five Horses Tavern (http://www.fivehorsestavern.com) for something a bit different.

File: mtucker01752-boston.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+## Boston, MA
+
+### Places to eat
+- Tavern in the Square - http://taverninthesquare.com
```
Comment (vicmaster) [inline mtucker01752-boston.md]: What about to use markdown for links like?:
- [Tavern in the Square](http://taverninthesquare.com)

What do you think?

Comment (markrtucker) [reply mtucker01752-boston.md]: @vicmaster - great idea! I edit the file locally, now to figure out how to make github recognize the changes....

**Review by vicmaster — APPROVED** [review]

**Review by vicmaster — APPROVED** [review]

**Review by vicmaster — APPROVED** [review]

File: mtucker01752-boston.md (hunk 1 above)
Comment (vicmaster) [reply mtucker01752-boston.md]: Done :)

**Comment by markrtucker** [comment]
Closes #11

**Pull request merged**